### PR TITLE
DOCS-1936: MDX2 compatibility updates

### PR DESCRIPTION
**Pull request by ctauchen**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: calico-cloud/reference/resources/packetcapture.mdx [hunk 1]
```diff
@@ -141,7 +141,7 @@ endTime: '2021-08-26T12:30:00Z'
...
 | directory | The path inside the calico-node container for the generated files.                                                                                                                                                                                                                                                |
-| fileNames | The name of the generated file for a `PacketCapture` ordered alphanumerically. <br/> The active packet capture file will be identified using the following schema: "{workload endpoint name}\_{host network interface}.pcap". <br/> Rotated capture files name will contain an index matching the rotation timestamp. |
+| fileNames | The name of the generated file for a `PacketCapture` ordered alphanumerically. <br/> The active packet capture file will be identified using the following schema: `{<workload-endpoint-name}\_{<host-network-interface>}.pcap`. <br/> Rotated capture files name will contain an index matching the rotation timestamp. |
```
Comment (ctauchen) [inline calico-cloud/reference/resources/packetcapture.mdx:144]: Is this right?

File: calico-enterprise/getting-started/upgrading/upgrading-calico-to-calico-enterprise/upgrade-to-tsee/helm.mdx [hunk 2]
```diff
@@ -46,16 +46,21 @@ The following steps assume the Calico deployment is installed on `tigera-operato
...
 
    <CodeBlock language='bash'>
      {'{{version}}' === 'master'
-       ? `helm upgrade calico-enterprise tigera-operator-v0.0.tgz \\
---set-file imagePullSecrets.tigera-pull-secret=<path/to/pull/secret>,tigera-prometheus-operator.imagePullSecrets.tigera-pull-secret=<path/to/pull/secret> \\
---namespace tigera-operator`
-       : `helm upgrade calico-enterprise tigera-operator-{{chart_version_name}}.tgz \\
---set-file imagePullSecrets.tigera-pull-secret=<path/to/pull/secret>,tigera-prometheus-operator.imagePullSecrets.tigera-pull-secret=<path/to/pull/secret> \\
---namespace tigera-operator`}
+       ? (
```
Comment (ctauchen) [inline calico-enterprise/getting-started/upgrading/upgrading-calico-to-calico-enterprise/upgrade-to-tsee/helm.mdx:53]: @ronanc-tigera The output is good, but does this change look okay to you?

File: calico-cloud/reference/resources/packetcapture.mdx (hunk 1 above)
Comment (bmckercher123) [reply calico-cloud/reference/resources/packetcapture.mdx:144]: Looks right to me.

**Review by bmckercher123 — APPROVED** [review]
LGTM. What a monster!

**Pull request merged**